org.adempiere.base/src/org/compiere/model/MUserDefField.java

```diff
@@ -37,6 +38,9 @@
 	 */
 	private static final long serialVersionUID = 2522038599257589829L;
 
+	/**	Cache of selected MUserDefField entries 					**/
+	private static CCache<String,MUserDefField> s_cache = new CCache<String,MUserDefField>(Table_Name, 10);
```

Comment:
please replace this with the new ImmutablePOCache (refer to MUserDefWin at master branch)